llvm/test/CodeGen/SPIRV/hlsl-intrinsics/refract.ll

```diff
@@ -0,0 +1,36 @@
+; RUN: llc -O0 -mtriple=spirv-unknown-vulkan %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv64-unknown-unknown %s -o - -filetype=obj | spirv-val %}
```

Comment:
There is no spirv64 target if you want to use the glsl extensions. Delete your second runline